TeachingRecordSystem/tests/TeachingRecordSystem.TestCommon/CrmTestData.CreatePerson.cs

```diff
@@ -16,15 +16,22 @@ public Task<CreatePersonResult> CreatePerson(Action<CreatePersonBuilder>? config
 
     public class CreatePersonBuilder
     {
+        private const string TeacherStatusQualifiedTeacherAssessmentOnlyRoute = "100";
```

Comment:
This is a little strange as a default - `71` - 'Qualified teacher (trained)' is probably better.

Reply:
I thought I'd changed that actually!

Reply:
Done